### DISCUSSION TITLE
Support Cosmosdb mongo

### DISCUSSION THREAD

**Discussion by alexisaraya** [Azure/data-api-builder#1342] [General]
Hi Guys,

Does the current version support cosmosdb with mongodb support ? 

Best Regards

**Comment by aaronburtle** [comment]
Hi @alexisaraya,

We do not currently have support for mongodb.

Cheers,
Aaron

**Comment by sajeetharan** [comment]
@alexisaraya @aaronburtle meant to say that we don't support Cosmos DB for MongoDB. Not native mongo! Hope that clarifies!

**Reply by opsingh861** [reply]
@sajeetharan will be in future we will get support for mongodb? 
It's almost end of 2024 so do we have mongodb support now?

**Reply by sajeetharan** [reply]
@opsingh861 Could you share more details on this request? Have you tried the NoSQL option? We don’t currently have plans to add this support.

**Reply by opsingh861** [reply]
@sajeetharan yeah I was trying nosql but right now it is only supporting cosmosdb I guess. The connection string which cosmosdb is expecting is different from the mongodb. Do you have any idea that how I can use mongodb in dab as db source? It is very crucial for my project.
